Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/05-Network-LZ/app-gateway.tf

```diff
@@ -1,5 +1,42 @@
 # Application Gateway and Supporting Infrastructure
 
+#############
+# LOCALS #
+#############
+
+/*
+
```

Comment:
Please add comment indicating that if Blue/Green scenario is not used then this code can be simplified by specifying only one app gateway.

Reply:
fixed